diff -Nru resin-cmp-1.0.4.orig/configure.in resin-cmp-1.0.4/configure.in
--- resin-cmp-1.0.4.orig/configure.in	Fri Dec 21 07:03:12 2001
+++ resin-cmp-1.0.4/configure.in	Fri Dec 21 07:05:12 2001
@@ -342,30 +342,12 @@
   AC_MSG_ERROR([Can't find Apache module directory ${apache_libexec}])
 fi
 
-#
-# Heuristics for finding the Apache configuration directory
-#
 if test -z "${apache_conf}" -a -n "${apxs}"; then
   conf_dir=`${apxs} -q SYSCONFDIR`
-
-  if test -r "${conf_dir}"/httpd.conf; then
-    apache_conf=${conf_dir}/httpd.conf
-  elif test -r "${conf_dir}"/apache.conf; then
-    apache_conf=${conf_dir}/apache.conf
-  elif test -r "${conf_dir}"/httpsd.conf; then
-    apache_conf=${conf_dir}/httpsd.conf
-  fi
-fi
-
-if test -z "${apache_conf}" -a -r "${apache_dir}/conf/httpd.conf"; then
-  apache_conf=${apache_dir}/conf/httpd.conf
-fi
-
-if test -z "${apache_conf}" -a -r "${apache_dir}/etc/httpd.conf"; then
-  apache_conf=${apache_dir}/etc/httpd.conf
+  apache_conf=${conf_dir}/httpd.conf
 fi
 
-if test -n "$apache" -a ! -r "${apache_conf}"; then
+if test -n "$apache" -a -z "${apache_conf}"; then
   AC_MSG_ERROR([Can't find Apache module configuration ${apache_conf}])
 fi
 #
