diff -Nru resin-cmp-1.0.4.orig/configure.in resin-cmp-1.0.4/configure.in
--- resin-cmp-1.0.4.orig/configure.in	Fri Dec 21 06:51:47 2001
+++ resin-cmp-1.0.4/configure.in	Fri Dec 21 06:57:19 2001
@@ -721,7 +721,7 @@
 ], [
 OPENSSL=${withval}
 
-if test ! -r "${OPENSSL}/lib/libssl.a"; then
+if test ! -r "${OPENSSL}/lib/libssl.so"; then
   AC_MSG_ERROR([Can't find valid OpenSSL ${openssl}])
 fi
 ])
@@ -737,7 +737,7 @@
 
 if test -z "${OPENSSL}"; then
   foo=bar
-elif test ! -r "${OPENSSL}/lib/libssl.a"; then
+elif test ! -r "${OPENSSL}/lib/libssl.so"; then
   AC_MSG_ERROR([Can't find valid OpenSSL in ${OPENSSL}])
 elif test ! -r "${OPENSSL}/include/openssl/ssl23.h"; then
   AC_MSG_ERROR([Can't find valid OpenSSL in ${OPENSSL}])
@@ -754,7 +754,7 @@
 
   INCLUDES="$INCLUDES -I$OPENSSL/include"
   SSL_OBJ=ssl.o
-  PROXY_LIBS="$PROXY_LIBS $OPENSSL/lib/libssl.a $OPENSSL/lib/libcrypto.a"
+  PROXY_LIBS="$PROXY_LIBS -lssl -lcrypto"
 fi
 
 AC_SUBST(INCLUDES)
